integration-tests/api-gateway/e2e/e2e-happy-path.test.ts

```diff
@@ -1,189 +1,136 @@
-import {
-  Payload,
-  getJarAuthorizationPayload,
-} from "../crypto/create-jar-request-payload";
 import {
   NINO,
-  CLIENT_ID,
   getClaimSet,
-  CLIENT_URL,
   environment,
+  testResourcesStack,
+  CLIENT_ID,
+  REDIRECT_URL,
+  AUDIENCE,
 } from "../env-variables";
-import { buildPrivateKeyJwtParams } from "../crypto/client";
 import { decodeJwt, JWK } from "jose";
 import {
   clearAttemptsTable,
   clearItemsFromTables,
 } from "../../resources/dynamodb-helper";
 import { stackOutputs } from "../../resources/cloudformation-helper";
+import {
+  authorizationEndpoint,
+  checkEndpoint,
+  createSession,
+  getJarAuthorization,
+} from "../endpoints";
+import { generatePrivateJwtParams } from "../crypto/private-key-jwt-helper";
 
-let data: any;
-let state: string;
-let authCode: any;
-let privateAPI: string;
-let publicAPI: string;
-let preOutput: Partial<{
-  PrivateApiGatewayId: string;
-  PublicApiGatewayId: string;
-}>;
-jest.setTimeout(30000);
-
-const createUpdatedClaimset = async (): Promise<any> => {
-  const updatedClaimset = await getClaimSet();
-  updatedClaimset.evidence_requested = {
-    scoringPolicy: "gpg45",
-    strengthScore: 2,
-  };
-  return updatedClaimset;
-};
-
-const createSessionId = async (
-  ipvCoreAuthorizationUrl: { client_id: any; request: string } | null
-): Promise<Response> => {
-  preOutput = await stackOutputs(process.env.STACK_NAME);
-  privateAPI = `${preOutput.PrivateApiGatewayId}`;
-  publicAPI = `${preOutput.PublicApiGatewayId}`;
-  const sessionApiUrl = `https://${privateAPI}.execute-api.eu-west-2.amazonaws.com/${environment}/session`;
-  const sessionResponse = await fetch(sessionApiUrl, {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
-      "X-Forwarded-For": "localhost",
-    },
-    body: JSON.stringify(ipvCoreAuthorizationUrl),
-  });
-  data = sessionResponse;
-  const session = await sessionResponse.json();
-  return session;
-};
+let sessionData: Response;
+let authCode: { value: string };
+let privateApi: string;
+let publicApi: string;
+
+jest.setTimeout(30_000);
 
 describe("End to end happy path journey", () => {
-  let session: any;
+  let state: string;
   let sessionId: string;
-  let publicEncryptionKeyBase64: string;
-  let privateSigningKey: JWK;
-  let personIdTableName: string;
-  let sessionTableName: string;
-  let audience: string;
+  let privateSigningKey: JWK | undefined;
+
   let output: Partial<{
     CommonStackName: string;
     StackName: string;
     PrivateApiGatewayId: string;
+    PublicApiGatewayId: string;
     NinoUsersTable: string;
     UserAttemptsTable: string;
   }>;
 
+  let commonStack: string;
+
   beforeAll(async () => {
-    audience = (await createUpdatedClaimset()).aud;
     output = await stackOutputs(process.env.STACK_NAME);
-    publicEncryptionKeyBase64 =
-      (await getSSMParameter(
-        "/check-hmrc-cri-api/test/publicEncryptionKeyBase64"
```

Comment:
Can this SSM param be removed now? It looks like it might have been clickopsed anyway